Frontend/src/lib/pages/LocationsPage.svelte

```diff
@@ -11,6 +15,8 @@
 </script>
 
 <main>
+    <PageHeader currentPage = {"Locations & Access Points"}/>
```

Comment:
probably best to have a variable for that

Reply:
For the sake of best practice? I feel like making a variable would just be unnecessarily writing the same thing twice.

Reply:
Ah I didn't realize that was coming from a specific page. It's still probably best practice to not use magic strings/values. Ig I would demote this to a nitpick, so this is fine if you keep it, but keep in mind that it is easier to manage variables in a "variables section" so to speak. My preference would actually be to have all the data for each page in a separate place to keep function and data totally apart, however that's probably just overengineering at this stage

Reply:
I will keep it for now. If it becomes too cluttered or "magical", I'll break it out.